javascript/javascript1/week4/appUsage.js

```diff
@@ -0,0 +1,54 @@
+
+//max usage of activity in day
+const usage = [];
+const usageNew = [];
+const durationArr = [];
+
+function addActivity(activity, duration) 
+{
+    const today = new Date();
+    usage.push({
+        date: today.toLocaleDateString("en-US"),
+        activity: activity,
+        duration: duration
+    })
+}
+addActivity("facebook", 10);
+addActivity("twitter", 15);
+addActivity("linked-in", 20);
+addActivity("youtube", 30);
+addActivity("twitter", 25);
+addActivity("whatsapp", 16);
+addActivity("youtube", 20);
+addActivity("twitter", 5);
+console.log(usage);
+
+function calculateActivities() 
+{
+    for (let i = 0; i < usage.length; i++) 
```

Comment:
consider creating a separate function for aggregation of usage.

Reply:
yes ,i made separate function for aggregateActivities.